androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -79,48 +111,68 @@ fun EnhancedVideoPlayer(
         }
     }
 
-    Box(
-        modifier = Modifier
-            .clickable(
-                indication = null,
-                interactionSource = remember { MutableInteractionSource() },
-                onClick = { isControlsVisible = !isControlsVisible }
+    AnimatedContent(
+        targetState = orientation,
+        transitionSpec = {
```

Comment:
Personally, I don't particularly appreciate how this animation is being handled. When the orientation changes, there is a second when two different frames are shown. Also, when we go to the landscape, there is a blank screen that the video slowly animates to occupy (slide right), which is far from optimal.

I'd go with how the YouTube app does the rotation. It is a simpler animation. Testing locally, removing this `transitionSpec` I could achieve a very similar behavior to the Youtube one:

[device-2023-06-05-140204.webm](https://github.com/profusion/android-enhanced-video-player/assets/11635469/43287ee8-f6be-425f-879e-0fcfe8958e82)